backend/index.js

```diff
@@ -20,17 +20,18 @@ const setupWebSecurity = require('./setupWebSecurity')
 const setupAuth = require('./setupAuth')
 const setupStaticContent = require('./setupStaticContent')
 const nunjucksSetup = require('./utils/nunjucksSetup')
-const setupWebpackForDev = require('./setupWebpackForDev')
 const setupRedirects = require('./setupRedirects')
 const setupApiRoutes = require('./setupApiRoutes')
 const setupReactRoutes = require('./setupReactRoutes')
 const phaseNameSetup = require('./phaseNameSetup')
-
 const pageNotFound = require('./setUpPageNotFound')
 
 const { logError } = require('./logError')
 const homepageController = require('./controllers/homepage/homepage')
 
+// eslint-disable-next-line  global-require
+const setupWebpackForDev = !config.app.disableWebpack && require('./setupWebpackForDev')
```

Comment:
There are no dev dependencies in the docker image. This allows us to bypass any unnecessary module imports.